docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -17,7 +17,9 @@ for image and container management. This article refers to the Device Mapper
 storage driver as `devicemapper`, and the kernel framework as `Device Mapper`.
 
 
->**Note**: The [Commercially Supported Docker Engine (CS-Engine) running on RHEL and CentOS Linux](https://www.docker.com/compatibility-maintenance) requires that you use the `devicemapper` storage driver.
+>**Note**: The [Commercially Supported Docker Engine (CS-Engine) running on RHEL
```

Comment:
Not in the procedure...the guideline is only in procedures to avoid distractions.